### ISSUE TITLE
0.5.0.beta1 does not run in Firefox/Webkit

### ISSUE THREAD

**Issue by wied03** [opal/opal-rspec#31]
When trying to run any spec, the following browsers/engines work (not exhaustive list):
* Chrome
* PhantomJS

The following do NOT:
* Firefox
* Headless webkit engine (using Capybara)

The latter 2 fail with a variant of this error when parsing RSpec code:
```
RuntimeError: Don't know how to parse $populate@http://127.0.0.1:42500/assets/rspec/core/metadata.self.js?body=1:102:47
```

**Comment by wied03** [comment]
Discovered while working on https://github.com/opal/opal-rails/pull/67

**Comment by elia** [comment]
Closed by #32
